arches_component_lab/src/arches_component_lab/generic/GenericFormField.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import { useTemplateRef } from "vue";
+
+import { FormField } from "@primevue/forms";
+import Message from "primevue/message";
+
+import type { FormFieldResolverOptions } from "@primevue/forms";
+
+const { nodeAlias, initialValue, validate, resolver } = defineProps<{
+    nodeAlias: string;
+    initialValue: unknown | null | undefined;
+    validate?: (event: unknown) => void;
+    resolver?: (event: FormFieldResolverOptions) => unknown | null | undefined;
```

Comment:
we should probably rename this to something more descriptive like `transformData` -- again in a followup